### ISSUE TITLE
Is it correct to  use cross entropy with att1 and att2 at same time for CelebA?

### ISSUE THREAD

**Issue by BossunWang** [sungho-CoolG/FSCL#2]
Hi expert ,
Nice work to implement contrastive learning on facial attribute.
I would like to let you know the BCEWITHLOGITSLOSS fit on pytorch when you using multi label classify.
https://github.com/sungho-CoolG/FSCL/blob/1369d1588f366533eae9bc7a65dd77783c1968e3/main_classifier.py#L166
